src/test/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/legalrepresentative/LegalRepresentativeRemissionDecisionRejectedPersonalisationTest.java

```diff
@@ -0,0 +1,71 @@
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCase;
+
+@ExtendWith(MockitoExtension.class)
+class LegalRepresentativeRemissionDecisionRejectedPersonalisationTest {
+
+    @Mock private AsylumCase asylumCase;
+
+    private String appealReferenceNumber = "someReferenceNumber";
+    private String legalRepRefNumber = "someLegalRepRefNumber";
+    private String appellantGivenNames = "someAppellantGivenNames";
+    private String appellantFamilyName = "someAppellantFamilyName";
+    private String iaExUiFrontendUrl = "http://somefrontendurl";
+    private String templateId = "someTemplateId";
+    private String legalRepEmailAddress = "legalRepEmailAddress@example.com";
+
+    private LegalRepresentativeRemissionDecisionRejectedPersonalisation legalRepresentativeRemissionDecisionRejectedPersonalisation;
+
+    @BeforeEach
+    void setUp() {
+
+        legalRepresentativeRemissionDecisionRejectedPersonalisation =
+            new LegalRepresentativeRemissionDecisionRejectedPersonalisation(templateId, iaExUiFrontendUrl);
+    }
+
+    @Test
+    void should_return_given_template_id() {
+        assertEquals(templateId, legalRepresentativeRemissionDecisionRejectedPersonalisation.getTemplateId(asylumCase));
+    }
+
+    @Test
+    void should_return_given_email_address_from_asylum_case() {
+        when(asylumCase.read(LEGAL_REPRESENTATIVE_EMAIL_ADDRESS, String.class))
+            .thenReturn(Optional.of(legalRepEmailAddress));
+
+        assertTrue(
+            legalRepresentativeRemissionDecisionRejectedPersonalisation.getRecipientsList(asylumCase).contains(legalRepEmailAddress));
+    }
+
+    @Test
+    void should_return_personalisation_when_all_information_given() {
+
+        when(asylumCase.read(APPEAL_REFERENCE_NUMBER, String.class)).thenReturn(Optional.of(appealReferenceNumber));
+
+        when(asylumCase.read(LEGAL_REP_REFERENCE_NUMBER, String.class)).thenReturn(Optional.of(legalRepRefNumber));
+        when(asylumCase.read(APPELLANT_GIVEN_NAMES, String.class)).thenReturn(Optional.of(appellantGivenNames));
+        when(asylumCase.read(APPELLANT_FAMILY_NAME, String.class)).thenReturn(Optional.of(appellantFamilyName));
+
+        Map<String, String> personalisation =
+            legalRepresentativeRemissionDecisionRejectedPersonalisation.getPersonalisation(asylumCase);
+
+        assertEquals(appealReferenceNumber, personalisation.get("appealReferenceNumber"));
+        assertEquals(legalRepRefNumber, personalisation.get("legalRepReferenceNumber"));
+        assertEquals(appellantGivenNames, personalisation.get("appellantGivenNames"));
+        assertEquals(appellantFamilyName, personalisation.get("appellantFamilyName"));
+        assertEquals(iaExUiFrontendUrl, personalisation.get("linkToOnlineService"));
+    }
```

Comment:
Similar to above, you will need to test for customer services telephone and email address.

Reply:
Added the telephoneNumber and emailAddress. Modified the template in Gov Notify